scripts/missions/windurst/9_1_Doll_of_the_Dead.lua

```diff
@@ -164,6 +164,7 @@ mission.sections =
             onEventFinish =
             {
                 [13] = function(player, csid, option, npc)
+                    player:tradeComplete()
```

Comment:
confirmTrade?

Reply:
Theoretically, since you're using tradeHasExactly; although I thought there was some effort to move from tradeComplete -> confirmTrade, but I could have made that up